test/distributed/org/apache/cassandra/distributed/test/cql3/StatefulASTBase.java

```diff
@@ -449,6 +563,21 @@ protected void compact()
             numCompact++;
         }
 
+        protected void repair()
+        {
+            if (mutationsSinceLastFlush > 0)
+                flush();
+
+            numRepairs++;
+            mutationsSinceLastRepair = 0;
+            flushesSinceLastRepair = 0;
+        }
+
+        protected boolean isConsistent()
+        {
+            return mutationsSinceLastRepair == 0;
```

Comment:
this isn't actually true... if you repair a single token then only that token is consistent...